### DISCUSSION TITLE
Ask for help

### DISCUSSION THREAD

**Discussion by qizaizs** [Chia-Network/chia-blockchain#9147] [Support]
Hello, the new agreement 130t has worked for more than ten days, and the income has suddenly disappeare
!There's only so little money left [1111111](https://user-images.githubusercontent.com/93752560/140455210-31f0f7cb-2751-4d39-b9fb-f5db517b287f.png)
![222222222222](https://user-images.githubusercontent.com/93752560/140455214-c3e9e4c7-13d0-421d-9004-70767b1579e8.png)
d

**Comment by loppefaaret** [comment]
did you send any coins ? do you see any outgoing transactions in your wallet history ? 
if noth of those questions are "no" - then you likely have a compromised mnemonic seed (24 words) and someone else is controlling your wallet from another machine, stealing your coins
